sat/cli/status/status_module.py

```diff
@@ -489,25 +489,24 @@ def get_image_for_component(self, raw_component):
         # this information anywhere else.
 
         ims_client = IMSClient(self.session)
-        booted_image = None
+        booted_image_id = None
 
         kernel_path = get_val_by_path(raw_component, 'actual_state.boot_artifacts.kernel')
         if kernel_path:
             img_id = urlparse(kernel_path).path.split('/')[1]
```

Comment:
At this point, we already have the IMS image ID without making an additional query to the IMS API. I propose that we stop here and just assume that we can trust BOS that this is the IMS image ID used to boot the node. Then we can get rid of the extra query to IMS and the extra possibility of warning messages when IMS images can't be found in IMS. This can occur when IMS images have been deleted, and such warning messages will probably just confuse the admin.